test/unit.js

```diff
@@ -0,0 +1,273 @@
+import camelCase from 'camel-case';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-addons-test-utils';
+import reactify from '../src/index';
+import skate from 'skatejs';
+import helperElement from 'skatejs/test/lib/element';
+
+describe('react-integration;', function () {
+    var fixture;
+
+    beforeEach(function () {
+        fixture = document.createElement('div');
```

Comment:
For example here, instead of assigning to `fixture`, you can create a scoped function to return the fixture element and use it within your test.